source/Tasks/TestBuildAcceptance.build.ps1

```diff
@@ -45,54 +41,51 @@ param
 )
 
 task TestBuildAcceptance {
-    $OutputDirectory = Get-SamplerAbsolutePath -Path $OutputDirectory -RelativeTo $ProjectPath
+    $PesterOutputFolder = Get-SamplerAbsolutePath -Path $PesterOutputFolder -RelativeTo $OutputDirectory
```

Comment:
You still need to get the absolute OutputDirectory before line 44 here.

Reply:
AFAIK, RequiredAssemblies makes sure that an an assembly exists but does not import it.

Reply:
This is done in line 11. `$OutputDirectory` is `Join-Path $BuildRoot 'output'`.

Reply:
Unfortunately that's not enough.
If a relative path is used as value, we don't know exactly what it will be relative to.
Best is to repeat the Get-SamplerAbsolutePath line for $OutputDirectory